services/brig/src/Brig/API/Client.hs

```diff
@@ -328,6 +329,52 @@ claimMultiPrekeyBundles protectee quc = do
       tUntagged . qualifyAs luc
         <$> claimLocalMultiPrekeyBundles protectee (tUnqualified luc)
 
+-- Similar to claimMultiPrekeyBundles except for the following changes
+-- 1) A new return type that contains both the client map and a list of
+--    users that messages couldn't be sent to.
```

Comment:
Perhaps a slightly more informative description would be "users that prekeys couldn't be fetched for."

Reply:
I've updated it with that text.